Kickstarter-iOS/Locales/de.lproj/Localizable.strings

```diff
@@ -322,6 +322,8 @@
 "Pledged_on_date" = "<b>Beitrag geleistet</b> am %{pledge_date}";
 "Popular" = "Beliebt";
 "Popular_Projects" = "Beliebte Projekte";
+"Post_code" = "Postleitzahl";
```

Comment:
Since these strings have been previously translated and their locale flavours were taken into account we now unfortunately have 3 strings that are all equal (in other than base localizations). I couldn't think of a way around this problem but am open to any ideas.